### PR TITLE
If command for report the note on the current Excel cell pressed twice, presents the information in browse mode

### PR DESCRIPTION
**Pull request by tseykovets**
### Link to issue number:

This is a minor change that improves one command for Microsoft Excel.

### Summary of the issue:

The text of cell notes often contains important voluminous information that is difficult to perceive in the mode of pronouncing a full phrase.

This pull request adds the ability to present the text of a note in browse mode by pressing command twice (NVDA+Alt+C).

### Description of user facing changes

The previously existing command for Microsoft Excel (NVDA+Alt+C) gained additional functionality when it is pressed twice.

### Description of development approach

* A handler for the number of presses has been added to the script (if repeats).
* If pressed twice, presents the information in browse mode (ui.browseableMessage).

### Testing strategy:

To test, just press the NVDA+Alkt+C command twice on the cell with the note.

The changes do not contain new code to interact with the operating system and Excel application.

### Known issues with pull request:

No issues known.
Since the change does not contain new code to interact with the operating system and Excel application, it is not expected to affect any other functionality of NVDA.

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [ ] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

**Comment by Adriani90** [comment]
I think this is not needed. Nvda has already its own handler for these notes and the nvda specific dialog with the coresponding note can be invoked with shift+f2 when focusing the relevant cell. See the MS Excel app module for more details.  In this proposed change is a downgrade to the current behavior and  most people will use shift+f2 because there you can even edit the note.

**Comment by cary-rowen** [comment]
Thank you very much, this is a feature I'm thinking of implementing, good job, please continue.
Please note, that there is a feature request #14628.
You can update the description of this PR.

**Comment by cary-rowen** [comment]
@Adriani90 
> I think this is not needed. Nvda has already its own handler for these notes and the nvda specific dialog with the coresponding note can be invoked with shift+f2 when focusing the relevant cell. See the MS Excel app module for more details. In this proposed change is a downgrade to the current behavior and most people will use shift+f2 because there you can even edit the note.

For consistency, if #15987 is acceptable, then this PR should also be accepted.

File: source/NVDAObjects/window/excel.py [hunk 1]
```diff
@@ -1679,8 +1679,11 @@ def _get_positionInfo(self):
 	# In Office 2016, 365 and newer, comments are now called notes.
 	# Thus, messages dialog title and so on should refer to notes.
 	@script(
-		# Translators: the description  for a script for Excel
-		description=_("Reports the note on the current cell"),
+		description=_(
+			# Translators: the description  for a script for Excel
```
Comment (seanbudd) [inline source/NVDAObjects/window/excel.py:1683]:
```suggestion
			# Translators: the description for a script for Excel
```

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -10,6 +10,9 @@ What's New in NVDA
 
 
 == Changes ==
+- Microsoft Office:
+  - When command for report the note on the current Excel cell (NVDA+Alt+C) pressed twice, presents the information in browse mode.
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:14]:
```suggestion
  - When the command to report the note of the current Excel cell is located is pressed twice  (``NVDA+alt+c``), the information is presented browse mode.
  -
```

File: user_docs/en/userGuide.t2t [hunk 3]
```diff
@@ -1255,7 +1255,7 @@ Selecting a form field and pressing enter or the Move to button moves to that fi
 
 +++ Reporting Notes +++[ExcelReportingComments]
 %kc:beginInclude
-To report any notes for the currently focused cell, press NVDA+alt+c.
+To report any notes for the currently focused cell, press NVDA+alt+c. Pressing twice shows the information in a window.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:1258]:
```suggestion
To report any notes for the currently focused cell, press ``NVDA+alt+c``.
Pressing twice shows the information in a window.
```

**Review by seanbudd — APPROVED** [review]

**Comment by seanbudd** [comment]
Please note the linting error, repeat is not definied

**Comment by seanbudd** [comment]
did you test this? i'm not sure how this code would succeed

**Comment by seanbudd** [comment]
Have you tested this? Is it ready for re-review? If so, mark the PR as "ready for review" so it is no longer in draft state.

**Comment by seanbudd** [comment]
have you tested this? please do not mark as ready for review if you have not tested this.

**Review by Qchristensen — APPROVED** [review]
User guide reads well.  The changes.txt text seems cumbersome to me - I would perhaps say:

 - Excel: When "report any notes for the currently focused cell" (``NVDA+alt+c``) is pressed twice, the information is shown in a window. (#15986, @tseykovets)

**Comment by Adriani90** [comment]
@tseykovets are you still working on this?

**Comment by tseykovets** [comment]
@Adriani90, it seems to me that I have implemented everything that is needed.
I would appreciate testing and information if something is wrong.

**Comment by hwf1324** [comment]
@tseykovets If everything is ready, switch the status of the PR from Draft to "ready for review". so that NV Access personnel can review your PR.

> When you think a contribution is ready, or you would like feedback, open a draft pull request. When you would like a review, mark the PR as "ready for review".

**Comment by seanbudd** [comment]
As requested twice previously, have you tested this? please do not mark as ready for review if you have not tested this.
You are expected to test your work when making a code contribution.

**Comment by Adriani90** [comment]
@tseykovets you need to solve the conflicts also in this PR, otherwise we cannot download a build with this PR to test.
Is it too difficult for you to test it yourself? Or what is the reason you didn't test it when raising this pull request?

**Comment by Adriani90** [comment]
@tseykovets are you still working on this? Or should we close this as abandoned?

**Comment by tseykovets** [comment]
At the end of 2023, I considered the implementation from this pull request to be complete and debugged.
However, I don't know how true this is for the current state of the master branch.
At the moment, I do not have the opportunity to quickly test the implementation, since I do not have a computer at hand with a configured environment for the local build of NVDA.
If someone undertakes to check the relevance of the implementation from this pull request, then I will be grateful for such help.

**Comment by XLTechie** [comment]
@tseykovets You can always perform your own AppVeyor builds of NVDA, in which case you do not need a local dev environment.

https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/buildingNVDAOnAppVeyor.md

**Comment by gerald-hartig** [comment]
As @seanbudd mentioned, PRs can only move from draft to ready status once testing has been successfully completed. Thorough testing is crucial to maintain the integrity and functionality of NVDA and is part of our Contribution Process (https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/contributing.md#overview-of-contribution-process). We strongly advise authors to test their own code but if the author is unable to do the testing, collaboration with someone in the community to carry out that testing is then essential. 

If for some reason before you start you know that you cannot carry out the testing, we advise you find someone who can do the testing before you write your first line of code. You can use the issue ticket for this purpose, or reach out to the development community in one of the user/developer groups.

For this PR, if no one is available to assist with the testing, we may need to consider closing this PR until testing can be completed.

**Comment by seanbudd** [comment]
Closing as abandoned

**Pull request closed without merging**